pages/dkp/kommander/1.4/release-notes/1.4.3/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: layout.pug
+beta: false
+navigationTitle: Release Notes Kommander 1.4.3
+title: Release Notes Kommander 1.4.3
+menuWeight: 40
+excerpt: View release-specific information for Kommander 1.4.3
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.3 was released on December 9, 2021.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/1.8/download/) and [install](/dkp/konvoy/1.8/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+## Features/Improvements
+
+- Updated Kubecost enterprise key
+
+### Bug fixes
+
+- Fixed an issue where Kommander UI runs out of ports. This monitors ports and forces a restart if it runs out. 
+
+- Fixed http proxing support in unidirectional environments (COPS-7078).
+
+- Updated Traefik Forward Auth to 3.0.3, pulling in a fix for URL pattern matching.
```

Comment:
Traefik Forward Auth is installed by Konvoy 1.x as part of the Kubeaddons, it's not part of Kommander 1.4.   This issue is already documented there:

https://github.com/mesosphere/konvoy/blob/release-1.8/docs/site/release-notes/kubernetes-base-addon/index.md?plain=1#L22

```suggestion
```